app/templates/suppliers/edit_what_buyers_will_see.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/forms/macros/forms.html" as forms %}
+
+{% block page_title %}What buyers will see – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account"
+      },
+      {
+        "link": url_for(".supplier_details"),
+        "label": "Company details"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if error %}
+<div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+  <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+    {{ error }}
+  </h3>
+</div>
+{% endif %}
+{% if supplier_form.errors or contact_form.errors %}
```

Comment:
You might be able to reduce this down to just using the validation macro. You could build a list of form errors in the view and pass it in? Like this: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/47-kk-registered-name-page/app/templates/suppliers/edit_registered_name.html#L29

Reply:
Hi @Wynndow we looked at it with @TheDoubleK and we decided against moving to macro at this moment for following reasons:
1. Since our validation uses two forms it might be a bit complex to implement.
2. We are short on time

I hope this is fine. Still, we changed the order of the errors, so summary validation shows after other validations, so it matches the order on the page.